src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -80,17 +83,26 @@ public void onBlockPlaceExisting(BlockPlaceEvent e) {
     public void onBlockPlace(BlockPlaceEvent e) {
         ItemStack item = e.getItemInHand();
         SlimefunItem sfItem = SlimefunItem.getByItem(item);
-
+        if (!Slimefun.instance().isUnitTest()) {
+            Slimefun.getProtectionManager().logAction(e.getPlayer(), e.getBlock(), Interaction.BREAK_BLOCK);
```

Comment:
Is this what fixed the issue? What was the cause?

Reply:
Yea i added a comment